Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
What is this supposed to fix? What is the failure, and how does this improve on it?

Look at `PACKAGES` is computed: it is supposed to be equivalent to `./...`, except that `vendor/**` is excluded. We don’t really want to run tests in the vendored packages, so I can’t see what difference this should make.